ml_perf/flags/9/train.flags

```diff
@@ -1,7 +1,12 @@
-# Flags for training.
+# train.flags: Flags for training.
 
 --flagfile=ml_perf/flags/9/architecture.flags
 
 --shuffle_buffer_size=10000
---filter_amount=0.1
---steps_to_train=256
+--filter_amount=0.1 
```

Comment:
Does filter_amount affect training result?   I saw it was 1.0 in Christian's repo, and it was 0.05 in v0.5 repo.   Not sure what it is doing.

Reply:
The AlphaGo Zero paper says it trains on moves sampled from the most recent 500,000 games; each training checkpoint is produced every 1,000 steps at batch size 2,048 = 2M examples.  500,000 games at average game length 200 = 100M examples.  So the original alphago zero paper suggests only using about 2% of the total examples!  But they have a lot more of them...

It's unclear what the correct value is for the much smaller mlperf scenario, so i don't know what the right thing to do is.  It seems like seeing every example about once is reasonable, which would imply setting filter_amount and window_size such that `filter_amount * window_size == 1`

Reply:
I see.  Does it mean if my learning rate works for filter_amount=1.0, and it is now changed to 0.1, I need to push learning rate to 10x to compenstate that I didn't see that sample 10 times?

Reply:
Probably not; most likely the value-head of the network is strongly overfit anyway due to the small number of games played.  To check, if the `value_cost` component of the training error is < 0.15, the value head is probably overfit.  The delta between the holdout and the train error is a good proxy for whether or not it's overfit.

I don't really know, though!  It's certainly worth trying.